system/utils.py

```diff
@@ -593,14 +701,14 @@ def get_pool_info(primary: str) -> dict:
     assert promote_res['op'] == 'REPLY'
 
 
-async def eventually_positive(func, *args, cycles_limit=15):
+async def eventually_positive(func, *args, cycles_limit=15, sleep=30, **kwargs):
```

Comment:
What about using same parameters (`retryTimeout` and `timeout`) as in `eventually`?